build.gradle

```diff
@@ -18,22 +22,39 @@ repositories {
 }
 
 def jacksonVersion = '2.10.2'
+def problemVersion = '0.24.0'
+def jupiterVersion = '5.5.2'
+def hamcrestVersion = '2.2'
 
 dependencies {
     implementation group: 'com.amazonaws', name: 'aws-lambda-java-core', version: '1.2.0'
-    implementation group: 'com.amazonaws', name: 'aws-java-sdk-lambda', version: '1.11.714'
+    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: jacksonVersion
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: jacksonVersion
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: jacksonVersion
     implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: jacksonVersion
-    implementation group: 'org.zalando', name: 'problem', version: '0.23.0'
-    implementation group: 'org.zalando', name: 'jackson-datatype-problem', version: '0.23.0'
+    implementation(group: 'org.zalando', name: 'problem', version: problemVersion) {
+        compileOnly(group: 'org.apiguardian', name: 'apiguardian-api', version: '1.1.0') {
+            because 'Dependency has maven scope provided'
+        }
+    }
+    implementation group: 'org.zalando', name: 'jackson-datatype-problem', version: problemVersion
+    implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.9'
+    implementation group: 'org.apache.httpcomponents', name: 'httpcore', version: '4.4.11'
 
-    implementation group:'com.github.BIBSYSDEV', name:'nva-doi-transformer', version:'0.1.3'
-    implementation group: 'com.github.BIBSYSDEV', name: 'nva-datamodel-java', version: 'v0.5.2'
+    implementation group: 'com.github.BIBSYSDEV', name: 'nva-doi-transformer', version: '0.1.5'
+    implementation group: 'com.github.BIBSYSDEV', name: 'nva-datamodel-java', version: '0.7.0'
 
-    testImplementation group:'com.github.BIBSYSDEV',name:'aws-build-tools', version:'v1.5.5'
-    testImplementation group:'org.hamcrest', name:'hamcrest', version:'2.2'
-    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.5.2'
+    testImplementation group: 'com.github.BIBSYSDEV', name: 'aws-build-tools', version: '1.5.5'
```

Comment:
do we need this library?

Reply:
Yes, it is in use in a test-helper, but the test-helper itself is unused. I have created a task to refactor this library in its entirety as there are a number of issues with it, rather than opening the can of worms.